src/mgr/DaemonServer.cc

```diff
@@ -2320,9 +2320,12 @@ void DaemonServer::_send_configure(ConnectionRef c)
 {
   assert(lock.is_locked_by_me());
 
+  OSDPerfMetricQuery query;
```

Comment:
@dillaman Here a query is added each time a configure message is sent (mainly for testing purpose), don't you think I should move it somewhere else (e.g. in a routine called upon reception of a perf query command ?)

Reply:
I'm fine w/ this for the initial version. Going forward, we can either make the ```MMgrConfigure``` a periodic message when queues are enabled in order to force the ```MMgrReport``` message, or we only send it upon change but then need the OSDs to periodically send the results.